instant/contract-deployments.md

```diff
@@ -2,21 +2,21 @@
-| Swap     | [`4.3.5`](https://www.npmjs.com/package/@airswap/swap/v/4.3.5)     | [`0xa03855b0695013cF331FbD91dd7cBc4C2211793f`](https://rinkeby.etherscan.io/address/0xa03855b0695013cF331FbD91dd7cBc4C2211793f) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
-| Indexer  | [`3.6.7`](https://www.npmjs.com/package/@airswap/indexer/v/3.6.7)  | [`0x10F6702447414cE1250Af5f7000D7c9A0f04E824`](https://rinkeby.etherscan.io/address/0x10F6702447414cE1250Af5f7000D7c9A0f04E824) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
-| Delegate | [`1.5.6`](https://www.npmjs.com/package/@airswap/delegate/v/1.5.6) | [`0xd393FB3f3062e208049FC75B26BFa5aB0890CBB3`](https://etherscan.io/address/0xd393FB3f3062e208049FC75B26BFa5aB0890CBB3)         | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
-| Wrapper  | [`2.5.7`](https://www.npmjs.com/package/@airswap/wrapper/v/2.5.7)  | [`0x33356FcCF3bdeb37f3CD3Dc5382Ae5BF4AAda26c`](https://rinkeby.etherscan.io/address/0x33356FcCF3bdeb37f3CD3Dc5382Ae5BF4AAda26c) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
-| Types    | [`2.4.4`](https://www.npmjs.com/package/@airswap/types/v/2.4.4)    | [`0xe25b7504856bfb230b7c32BC87047479815cbc70`](https://rinkeby.etherscan.io/address/0xe25b7504856bfb230b7c32BC87047479815cbc70) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
+| Swap     | [`5.4.7`](https://www.npmjs.com/package/@airswap/swap/v/5.4.7)     | [`0xa03855b0695013cF331FbD91dd7cBc4C2211793f`](https://rinkeby.etherscan.io/address/0xa03855b0695013cF331FbD91dd7cBc4C2211793f) | [`953956f`](https://github.com/airswap/airswap-protocols/commit/953956f308c65ec53d1f1b20d35f47fe04b936af) |
+| Indexer  | [`3.6.8`](https://www.npmjs.com/package/@airswap/indexer/v/3.6.8)  | [`0x10F6702447414cE1250Af5f7000D7c9A0f04E824`](https://rinkeby.etherscan.io/address/0x10F6702447414cE1250Af5f7000D7c9A0f04E824) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
+| Delegate | [`2.6.8`](https://www.npmjs.com/package/@airswap/delegate/v/2.6.8) | [`0xd393FB3f3062e208049FC75B26BFa5aB0890CBB3`](https://etherscan.io/address/0xd393FB3f3062e208049FC75B26BFa5aB0890CBB3)         | [`953956f`](https://github.com/airswap/airswap-protocols/commit/953956f308c65ec53d1f1b20d35f47fe04b936af) |
+| Wrapper  | [`3.6.8`](https://www.npmjs.com/package/@airswap/wrapper/v/3.6.8)  | [`0x33356FcCF3bdeb37f3CD3Dc5382Ae5BF4AAda26c`](https://rinkeby.etherscan.io/address/0x33356FcCF3bdeb37f3CD3Dc5382Ae5BF4AAda26c) | [`953956f`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
```

Comment:
github link doesnt match the displayed commit

Reply:
nice catch!